src/main/java/com/maplexpbar/MapleXPBarPlugin.java

```diff
@@ -73,6 +81,12 @@ MapleXPBarConfig provideConfig(ConfigManager configManager)
 	{
 		return configManager.getConfig(MapleXPBarConfig.class);
 	}
+
+	@Subscribe
+	public void onStatChanged(StatChanged statChanged) {
```

Comment:
Unfortunately this also seems to trigger when a stat changes without xp being given, such as healing, regenerating HP, prayer drain, or boosted/drained skills. We'll need to find something specific to xp being given to a skill to avoid the bar changing unintentionally.